src/interpreter.jl

```diff
@@ -65,6 +65,13 @@ end
 #=CC.=#get_inference_world(interp::CthulhuInterpreter) = get_inference_world(interp.native)
 CC.get_inference_cache(interp::CthulhuInterpreter) = get_inference_cache(interp.native)
 
+struct CthulhuCacheToken
+    native::Any
+end
+@static if isdefined(CC, :cache_owner)
+    CC.cache_owner(interp::CthulhuInterpreter) = CthulhuCacheToken(CC.cache_owner(interp.native))
```

Comment:
We could in theory also use the cache of the AbsInt we are wrapping, but that seems dangerous.